package.json

```diff
@@ -127,7 +127,7 @@
     "babel-plugin-ember-template-compilation": "^2.1.1",
     "dag-map": "^2.0.2",
     "decorator-transforms": "2.0.0",
-    "ember-cli": "^4.10.0",
+    "ember-cli": "^6.1.0",
```

Comment:
This actually installed v6.2.2 due to the `^`so I should just list that version instead or use `~` instead.